openwrt/files/etc/udhcpc.user

```diff
@@ -1,19 +1,33 @@
 #!/bin/sh
 
+set -x
+
 case "$1" in
   # Same actions for renew or bound for the time being
   "renew"|"bound")
     # dump params to tmp so its easier to troubleshoot
     set > /tmp/dhcp.params
     radio0=`uci show 'wireless.radio0.channel'|cut -f 2 -d "'"`
     radio1=`uci show 'wireless.radio1.channel'|cut -f 2 -d "'"`
+
     if [ ! -z "$opt224" ] || [ ! -z "$opt225" ]; then
```

Comment:
a separate bug I just noticed, if we send one of the dhcp options but not the other, it attempts to set the other one to null, the || should be changed to && to require that both be sent.